packages/terra-time-input/src/TimeUtil.js

```diff
@@ -436,6 +436,8 @@ TimeUtil.DoubleZeroDigit = '00';
 
 TimeUtil.initialValue = ['0', '00'];
 
+TimeUtil.letterKeycode = [38, 40, 65, 66, 67, 68, 69, 70, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90];
```

Comment:
What is this array for?

Reply:
Safari browser allows letters when we have type as number, to prevent this we have to check all the letters keyCode

Reply:
Can you use regex like this.
```
        let inputValue = e.target.value;
        let numericValue = inputValue.replace(/[^0-9]/g, '');
        e.target.value = numericValue;
```

Reply:
handleKeyDown event is triggering when we press any letter, so i have added regex check in keyDown event and removed keyCode array value from TimeUtil file